### ISSUE TITLE
video height is cropping when it used in a modal | No autoResize Option

### ISSUE THREAD

**Issue by thilinadinith** [queenvictoria/ember-videojs#2]
I have replaced ivy videojs wthi ember videojs and im opening the video in a modal view. 
the video is not adjusting to the height of the modal where i cannot see the controlls and its cropping.
how can i fix this ?

**Comment by queenvictoria** [comment]
Yeah I don't think I am passing though the `fluid` tag. Can you show me your htmlbars component please?

```
{{video-js ...}}
```
